testing/internal/e2e/tests/static_with_vault/credential_store_test.go

```diff
@@ -138,14 +165,26 @@ func TestCliVaultCredentialStore(t *testing.T) {
 	require.NoError(t, err)
 
 	newSessionAuthorization := newSessionAuthorizationResult.Item
-	retrievedUser := fmt.Sprintf("%s", newSessionAuthorization.Credentials[0].Credential["username"])
-	retrievedKey := fmt.Sprintf("%s", newSessionAuthorization.Credentials[0].Credential["private_key"])
-	assert.Equal(t, c.TargetSshUser, retrievedUser)
+	require.Len(t, newSessionAuthorization.Credentials, 2)
 
-	k, err := os.ReadFile(c.TargetSshKeyPath)
-	require.NoError(t, err)
-	require.Equal(t, string(k), retrievedKey)
-	t.Log("Successfully retrieved credentials for target")
+	for _, v := range newSessionAuthorization.Credentials {
+		if v.CredentialSource.CredentialType == "ssh_private_key" {
+			retrievedUser := newSessionAuthorization.Credentials[0].Credential["username"].(string)
```

Comment:
If you're going to use a loop, use the loop variable too 😛 
```suggestion
			retrievedUser := v.Credential["username"].(string)
```

Reply:
Oops